sdk/src/testUtils.ts

```diff
@@ -46,3 +47,15 @@ export const getEnvTestAccount = async (keyring: Keyring): Promise<IKeyringPair>
 
   return keyring.addFromUri(accountUri)
 }
+
+export const toSqrtPrice = (x: bigint, decimals: bigint = 0n): SqrtPrice => {
```

Comment:
Well, we need to repeat this for all types. There is already equivalent of this function in rust `from_decimal`. Please add this function in wasm scale macro. Same as here: https://github.com/invariant-labs/protocol-a0/pull/167/files#r1451898576